.github/workflows/release.yml

```diff
@@ -55,12 +58,27 @@ jobs:
         run: |
           git pull
           npm install
-      - name: Semantic Release runs
+      - name: Semantic Release runs for draft release
+        if: ${{github.event.inputs.draft-release == 'true'}}
         id: semantic-release
+        run: npx semantic-release -t \${version}-draft
+        env:
+          GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
Are we planning to push the draft release tag also to main npmjs? and draft tag will be released as normal release.